qt/cmake-project/cmake-project

```diff
@@ -50,19 +56,30 @@ if [ "$QTVERSION" != "none" ]; then
     else
         QT=Qt$QTVERSION
     fi
-    QTLIBS="Core Gui"
-    QTTARGETS="$QT::Core $QT::Gui"
+    QTLIBS="Core Gui Widgets"
+    QTTARGETS="$QT::Core $QT::Gui $QT::Widgets"
     if [ -n "`grep QApplication *.cpp`" ]; then
         QTLIBS="$QTLIBS Widgets"
         QTTARGETS="$QTTARGETS $QT::Widgets"
         AUTOUIC="set(CMAKE_AUTOUIC TRUE)"
     fi
 fi
 
+if [ ! -d "cmake" ]; then
+  mkdir cmake
+fi
+SCRIPT=$(dirname -- "$(readlink -f "${BASH_SOURCE}")")
```

Comment:
This bash-ism won't work with other shells. Why not just 

```
SCRIPTDIR=`dirname $0`
```

?
(I added DIR for clarity)

Reply:
I changed it to your version.